daemons/controld/pacemaker-controld.h

```diff
@@ -1,26 +1,22 @@
-/* 
- * Copyright (C) 2004 Andrew Beekhof <andrew@beekhof.net>
- * 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+/*
+ * Copyright 2004-2018 Andrew Beekhof <andrew@beekhof.net>
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
```

Comment:
Should be GPLv2+ since this is not a library, right?

Reply:
Sure, but I just wanted to update the notice to current style standards while updating the date, not re-license it (though I did switch from 2 to 2.1 since that was permitted by the original). The file is trivial and not particularly useful on its own, so I think it's academic.

Reply:
Matter of consistency with declared intention.

See another example authored by you:
https://github.com/ClusterLabs/pacemaker/commit/599d6cc8dd535ecb658bb745275d72e7b00193e8

Why to bother with a cosmetics only midstep?

Reply:
That time, I had confirmed with the original licensor (Beekhof) what the intent was before changing. I didn't bother this time b/c there's no harm in licensing this file LGPL. Whenever I need to update the copyright notice (for the date in this case), I bring it into the current style standard.